openedx_events/tooling.py

```diff
@@ -147,9 +153,17 @@ def validate_sender():
 
         kwargs["metadata"] = self.generate_signal_metadata()
 
-        if send_robust:
-            return super().send_robust(sender=None, **kwargs)
-        return super().send(sender=None, **kwargs)
+        if self._allow_send_event_failure or settings.DEBUG:
+            return super().send(sender=None, **kwargs)
+
+        responses = super().send_robust(sender=None, **kwargs)
+        log.info(
+            "Responses of the Open edX Event <%s>: \n%s",
+            self.event_type,
+            pformat(responses, depth=2),
```

Comment:
From the image you left in the cover letter I get that we would print something like:

`function enrollment_receiver at 0xAAAAAA, Exception()`

for each response.

Can we make it log the stacktrace in case there is an exception?
Also, can we print the package or full path of the function? logging the function object does not give great information for debug

Reply:
This was what I managed to do:
![log-event-responses](https://user-images.githubusercontent.com/64440265/133318468-a446d703-c8dc-48eb-b547-14d3081b5ee9.png)
What do you think?